internals/set-parcel-build-variables.js

```diff
@@ -0,0 +1,11 @@
+const buildEnv = Object.create(process.env)
+
+const { buildTime, sha } = require('./getBuildData.js')
+const { csp } = require('./getCsp.js')
+
+buildEnv.REACT_APP_BYPASS_LOCAL = 1
```

Comment:
(This could go into `.env.production`)

(And we can simplify `process.env.NODE_ENV && process.env.NODE_ENV !== 'production' && !process.env.REACT_APP_BYPASS_LOCAL` checks to `!process.env.REACT_APP_BYPASS_LOCAL`)

Reply:
oopos thanks, I left it here by accident actually

Reply:
I think I don't want to have `REACT_APP_BYPASS_LOCAL` env at all here and especially in env.production. 

side note note related to Parcel migration: I would actually remove these conditions in `networkSaga` and `initialState` completely one day to limit build artefacts differences between envs.

Reply:
We can make default behaviour equal to REACT_APP_BYPASS_LOCAL=1, but some tests run against local network - don't think we can avoid different build

Reply:
Yep at least in one case it will exist in out code where we add an item to a network dropdown. Anyway, can we keep behaviour the same compared to current build for now?